### PR TITLE
Added Windows instructions - tried to make them as simple as possible, I hope this helps someone

### PR DESCRIPTION
**Pull request by tomhoward87**

### CODE REVIEW

File: README_for_Windows.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+5. Click the dropdown in the "New" column, change it from "Skip" to "20.1.1-1"
+6. Hit next to complete the installation.
+7. Run Cygwin by going to C:\cygwin64\bin (assuming default install location) and doubleclicking on "mintty.exe" to open a terminal
+8a. If you're comfortable with Git, clone the repo into the home directory. 
+8b. If you don't know what that means, in the Windows File Manager, go to 
+"C:\cygwin64\home\<your username here>" and right click on empty space, choose "New" and then Text Document. Copy and paste the contents of the file https://github.com/robintw/ParentZonePhotoDownloader/blob/master/download_parentzone_photos.py
+into the text document. Save the document as "download_parentzone_photos.py"
+9. Download ChromeDriver from here: https://chromedriver.chromium.org/downloads (choose the link matching the version of Chrome you have installed, and the file ending in "win32.zip")
+10. Extract chromedriver.exe from the zip file, and copy it into "C:\cygwin64\home\<your username here>" in the Windows File Manager.
+11. In the Cygwin terminal, run this command "export PATH=$^CTH:$(pwd)"
```
Comment (robintw) [inline README_for_Windows.md]: Is this a typo? The `^CTH` bit looks a bit odd.

Comment (tomhoward87) [reply README_for_Windows.md]: Whoops, yes, it should be "export PATH=$PATH:$(pwd)"

Comment (tomhoward87) [reply README_for_Windows.md]: I'll update the PR when I'm next at my PC.

**Comment by robintw** [comment]
Looks great - thanks for the contribution!

**Pull request merged**